server/src/test/resources/sql/table.iq

```diff
@@ -50,6 +50,21 @@ select * from t;
 
 !ok
 
+
+truncate table t;
```

Comment:
`TRUNCATE` is DDL like `DROP`, so `0 rows modified`.

Reply:
Good to know, I also tried in MySQL and confirmed your statement.